msteams-platform/tabs/how-to/authentication/auth-flow-tab.md

```diff
@@ -6,42 +6,48 @@ keywords: teams authentication flow tabs
 ---
 # Microsoft Teams authentication flow for tabs
 
-> [!Note]
-> For authentication to work for your tab on mobile clients, you need to ensure you're using at least the 1.4.1 version of the Teams JavaScript SDK.
-> Teams SDK launches separate window for authentication flow and hence samesite attribute can be set to 'Lax'. Currently, SameSite=None is not supported by the Teams desktop client or older versions of Chrome or Safari.
+> [!NOTE]
+> For authentication to work for your tab on mobile clients, you need to ensure you are using at least 1.4.1 version of the Microsoft Teams JavaScript SDK.
+> Teams SDK launches separate window for authentication flow. samesite attribute can be set to Lax. Teams desktop client or older versions of Chrome or Safari do not support SameSite=None.
```

Comment:
Please confirm on the convention of the attribute `samesite` or `SameSite` and update accordingly.
```suggestion
> [!NOTE]
> For authentication to work for your tab on mobile clients, you must ensure you are using at least 1.4.1 version of the Microsoft Teams JavaScript SDK.
> Teams SDK launches separate window for authentication flow. Set the `samesite` attribute to *Lax*. Teams desktop client or older versions of Chrome or Safari do not support `SameSite`=None.
```